recipe/meta.yaml

```diff
@@ -38,14 +38,13 @@ requirements:
     - cython
     - setuptools_scm
     - extension-helpers
-    - numpy 1.18.*  # [py<39 and build_platform == target_platform and python_impl == 'cpython']
+    - numpy 1.17.*  # [py<39 and build_platform == target_platform and python_impl == 'cpython']
     - numpy         # [py>=39 or build_platform != target_platform or  python_impl == 'pypy']
   run:
     - python
     - {{ pin_compatible("numpy") }}
-    - numpy >=1.18
-    - pyerfa >=2.0
-    - pyyaml >=3.13
+    - numpy >=1.17
```

Comment:
```suggestion
```

Reply:
Thanks @isuruf -- I'm confused though, where does the minimum numpy version get enforced at a package level if not here? I understand that conda-forge sets minimum numpy versions for each version of python already, I just always seem to get this wrong...

Reply:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy